pytorch_lightning/utilities/apply_func.py

```diff
@@ -35,6 +35,9 @@
     Batch = type(None)
 
 
+CPU_DEVICES = (None, "cpu", torch.device("cpu"))
```

Comment:
the device can't be None, see the type annotation below.

Reply:
My first attempt at fixing this was basically `torch.device(device) == torch.device("cpu")`, and it failed unit tests because we get None here, contrary to the type annotation.

That said, we can remove `None` here, since that case is a no-op (`torch.to(None, non_blocking=True)` is safe, and does nothing).

Reply:
I've removed `None`, so I think this can be resolved.